packages/lambda-xyz/src/tile.set.ts

```diff
@@ -17,8 +17,8 @@ export class TileSet {
     name: string;
     tag: TileSetTag;
     projection: EPSG;
-    tileSet: TileMetadataSetRecord;
-    imagery: Map<string, TileMetadataImageryRecord>;
+    private tileSet: TileMetadataSetRecord;
```

Comment:
forcing these private so people will be forced to use `allImagery()` which provides a sorted rule set